source/security/private-endpoints.txt

```diff
@@ -214,21 +188,161 @@ connection using either the App Services UI or the Admin API.
+                     ...
+                     ]
+            #. Get the ``service_name`` field from the desired entry in the
+               previous step.
+            #. Create a VPC Endpoint using the AWS CLI `create-vpc-endpoint 
+               <https://docs.aws.amazon.com/cli/latest/reference/ec2/create-vpc-endpoint.html>`__
+               command. You need an App Services endpoint service name, VPC ID,
+               subnet IDs, and AWS deployment region. The command should look
+               something like this:
+                  
             .. code-block:: sh
 
-               aws ec2 modify-vpc-endpoint \
-                 --region <your-aws-deployment-region> \
-                 --vpc-endpoint-id <your-vpc-endpoint-id> \
-                 --private-dns-enabled
+              aws ec2 create-vpc-endpoint \
+                --vpc-endpoint-type Interface
+                --service-name <App Services Endpoint Service Name> \
+                --vpc-id <your-vpc-id> \
+                --region <your-aws-deployment-region> \
+                --subnet-ids <your-subnet-ids>
+
+         .. step:: Add the Private Endpoint to Your App
+
+            Now that you've created the interface endpoint in your VPC, you
+            need to add it to your App. 
+
+            a. Find the Sync **VPC Endpoint ID**, either in the
+               output of the CLI commands you ran in the previous step or on the
+               endpoint details screen of the AWS VPC dashboard. This value
+               starts with ``vpce-``.
+            #. Create a Sync Private Endpoint by calling
+               :admin-api-endpoint:`Create a VPC Private Endpoint <operation/adminCreatePrivateEndpoint>`
+               with its Endpoint ID, cloud deployment region, and endpoint type.
+               For example, using ``aws-us-east-1``:
+
+               .. code-block:: sh
+
+                  curl https://services.cloud.mongodb.com/api/admin/v3.0/groups/{groupId}/apps/{appId}/security/private_endpoints \
+                  -X POST \
+                  -H "Authorization: Bearer <accessToken>" \
+                  -H "Content-Type: application/json" \
+                  -d '{
+                     "cloud_provider_region": "<your-cloud-deployment-region>",
+                     "cloud_provider_endpoint_id": "<your-vpc-endpoint-id>",
+                     "endpoint_type": "sync",
+                     "comment": "<comment-about-endpoint>"
+                  }'
+            #. You must also create a General Private Endpoint by following the
+               steps for creating a General Private Endpoint on this page.
+
+Migrate a Legacy Endpoint
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+If you have an endpoint that uses the legacy ``realm.mongodb.com`` domain, you
+must migrate it to use the new ``services.cloud.mongodb.com`` domain. You can't
+create new legacy endpoints.
+
+**To migrate a Legacy Endpoint:** 
+
+#. Create new endpoints to replace existing VPC endpoints
```

Comment:
For this step, do we mean users must go through the entire process to create new General or Sync endpoints - i.e. create new VPC Interface Endpoints and add them to App Services as a General or Sync Private Endpoint? If so, I wonder if we want to be more explicit? Maybe something like:

> Following the procedures on this page, create new General or Sync VPC Endpoints and add them to your App Services App.